src/pages/base/markets/Market.js

```diff
@@ -92,6 +94,16 @@ function Market(props) {
         isTokenCurrencyTotal={isTokenCurrencyTotal}
         className="mb-4"
       />
+      <div>
+        <div className="row">
+          <div className="col-xs-6">
+            <Top5Supply symbol={symbol} />
```

Comment:
You don't need to create Top5Borrow.js and Top5Supply.js. It should be the same table. Also why would we make 2 calls to api if we didn't need to? The way it is now, Top5Borrow calls api and gets supply data for no reason and Top5Supply does the same and get's borrow data for no reason.